src/main/java/senla/com/socialnetwork/security/jwt/JwtTokenUtils.java

```diff
@@ -0,0 +1,66 @@
+package senla.com.socialnetwork.security.jwt;
+
+import io.jsonwebtoken.*;
```

Comment:
Don't use * imports.  It's a holy war question, but you can read why it can be a bad thing in big projects when you have a lot of dependencies.